bsp/resources/META-INF/BSP.xml

```diff
@@ -5,6 +5,8 @@
     <extensionPoints>
         <extensionPoint qualifiedName="com.intellij.bspEnvironmentRunnerExtension"
                         interface="org.jetbrains.bsp.project.test.environment.BspEnvironmentRunnerExtension"/>
+        <extensionPoint qualifiedName="com.intellij.bspResolverNamingExtension"
```

Comment:
Going forward, all extension points should be declared dynamic so that the Scala plugin can become dynamic.

See https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_extension_points.html#dynamic-
extension-points

https://youtrack.jetbrains.com/issue/SCL-17035